Q/Qhull/build_tarballs.jl

```diff
@@ -0,0 +1,86 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "Qhull"
+version = v"2019.1"
+
+# Collection of sources required to build FFMPEG
+sources = [
+    "https://github.com/qhull/qhull/archive/$(version.major).$(version.minor).tar.gz" =>
+    "cf7235b76244595a86b9407b906e3259502b744528318f2178155e5899d6cf9f",
+]
+
+# Bash recipe for building across all platforms
+# TODO: Theora once it's available
+script = raw"""
+# initial setup
+cd $WORKSPACE/srcdir
+cd Qhull/
+apk add coreutils yasm
+
+if [[ "${target}" == *-linux-* ]]; then
+    export ccOS="linux"
+elif [[ "${target}" == *-apple-* ]]; then
+    export ccOS="darwin"
+elif [[ "${target}" == *-w32-* ]]; then
+    export ccOS="mingw32"
+elif [[ "${target}" == *-w64-* ]]; then
+    export ccOS="mingw64"
+elif [[ "${target}" == *-unknown-freebsd* ]]; then
+    export ccOS="freebsd"
+else
+    export ccOS="linux"
+fi
+
+if [[ "${target}" == x86_64-* ]]; then
+    export ccARCH="x86_64"
+elif [[ "${target}" == i686-* ]]; then
+    export ccARCH="i686"
+elif [[ "${target}" == arm-* ]]; then
+    export ccARCH="arm"
+elif [[ "${target}" == aarch64-* ]]; then
+    export ccARCH="aarch64"
+elif [[ "${target}" == powerpc64le-* ]]; then
+    export ccARCH="powerpc64le"
+else
+    export ccARCH="x86_64"
+fi
+
+pkg-config --list-all
+
+# begin the build process
+
+cd build
+# Generate makefiles
+cmake -G "Unix Makefiles" .. && cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ..
+# Ensure the config is correct
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ..
+# Run the build script
+make
```

Comment:
Let's save some time by building in parallel
```suggestion
make -j${nproc}
```